ports/poco/portfile.cmake

```
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO pocoproject/poco
     REF "poco-${VERSION}-release"
```

```suggestion
    REF "poco-1.12.5p1-release"
```
Please modify to resolve the following errors:
```
Downloading https://vcpkgassetcacheeastasia.blob.core.windows.net/cache/9cf885f67d87b86d3d5337aa2a3247a5828e071bba04304dfb79f7e00d94cd245e5def5191400a5753eebe0611b7f120c470cb0e0ab9b120f1d8b5f202d7d3d7?*** SECRET ***
Downloading https://github.com/pocoproject/poco/archive/poco-1.12.5.1-release.tar.gz
error: Failed to download from mirror set
error: https://vcpkgassetcacheeastasia.blob.core.windows.net/cache/9cf885f67d87b86d3d5337aa2a3247a5828e071bba04304dfb79f7e00d94cd245e5def5191400a5753eebe0611b7f120c470cb0e0ab9b120f1d8b5f202d7d3d7?*** SECRET ***: failed: status code 404
error: https://github.com/pocoproject/poco/archive/poco-1.12.5.1-release.tar.gz: failed: status code 404
```

---

This is what I meant by

> the REF in the portfile must be modified to use 1.12.5p1 literally.

Remember to mark the PR as "Ready for review" again.

---

@dg0yt ahh, yes makes sense now. Thanks again.